src/openstudio_lib/CMakeLists.txt

```diff
@@ -707,6 +707,19 @@ set(${target_name}_moc
 
 )
 
+set(TS_FILES
+  ${PROJECT_SOURCE_DIR}/translations/OpenStudioApp_fr.ts
+)
+#set_source_files_properties(${TS_FILES} PROPERTIES OUTPUT_LOCATION $<TARGET_FILE_DIR:${target_name}>/translations)
+
+qt5_create_translation(QM_FILES
+  ${PROJECT_SOURCE_DIR}
+  ${${target_name}_SRC}
+  ${TS_FILES})
+#include_directories(SYSTEM ${RUBY_INCLUDE_DIRS})
+
+add_custom_target(translations ALL DEPENDS ${QM_FILES})
```

Comment:
Are `QM_FILES` compiled in to the app or installed as a resource?

Reply:
Initially I was embeddeding them in the `openstudio.qrc`, but it's more trouble that needed, and now that I carry over some of the Qt internal ones, I figured we'd just install them where Qt expects them, that is `PREFIX_PATH/bin/translations` (Content/MacOS/blablalba on mac, I haven't played much with this)

Reply:
see `qt.conf` file too.